components/SocialMediaConnect.vue

```diff
@@ -294,9 +294,9 @@ export default {
       if (!isConnected) {
         this.connect(socialMedia);
       } else {
-        const url = this.getSocialMediaUrl(socialMedia);
+        const { url } = platform;
```

Comment:
where is `getSocialMediaUrl` used now? what is it actual function is not used now

Reply:
updated